### PR TITLE
add: eva icons

### LINKED ISSUE #1792: [Add] Eva Icons
### Site URL

https://akveo.github.io/eva-icons/

### Description of the above provided site

Eva Icons is a pack of more than 480 beautifully crafted Open Source icons for common actions and items.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by akshayitzme**
## Screenshots

image

fixes #1792

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @akshayitzme! :) Welcome to LinksHub! :) Your PR has been closed because you have not raised an issue for this link. Please read the Issue Requests section of our Contributing Guide to learn how to do this. After your raised issue is approved and assigned to you, your PR will be reopened.

**Comment by akshayitzme** [comment]
fixes #1792 

@CBID2 please reopen this PR

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me.

File: database/frontend/icons.json [hunk 1]
```diff
@@ -202,11 +202,18 @@
...
+    "subcategory": "icons"
+  }
   {
     "name": "Remix Icon",
     "description": "Remix Icon is a website that provides free icons for both personal and commercial use",
     "url": "https://remixicon.com/",
     "category": "frontend",
     "subcategory": "icons"
-  }
+  },
```
Comment (Anmol-Baranwal) [inline database/frontend/icons.json:218]:
```suggestion
  }
```

File: database/frontend/icons.json [hunk 2]
```diff
@@ -202,11 +202,18 @@
     "category": "frontend",
     "subcategory": "icons"
   },
+  {
+    "name": "Eva Icons",
+    "description": "Eva Icons is a pack of more than 480 beautifully crafted Open Source icons for common actions and items.",
+    "url": "https://akveo.github.io/eva-icons/",
+    "category": "frontend",
+    "subcategory": "icons"
+  }
```
Comment (Anmol-Baranwal) [inline database/frontend/icons.json:211]:
```suggestion
  },
```

**Review by Anmol-Baranwal — COMMENTED** [review]
Don't worry, I mistakenly did this when I was resolving the merge conflict.

**Comment by Anmol-Baranwal** [comment]
Done! 😅

**Comment by Anmol-Baranwal** [comment]
@CBID2 
review and merge it.

**Review by CBID2 — APPROVED** [review]
Well done @akshayitzme! :) You're making great progress! :) Keep it up! :)

**Comment by CBID2** [comment]
> @CBID2 
> 
> review and merge it.

Lol, I need your review first

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by CBID2** [comment]
Well done @akshayitzme! :)

**Pull request merged**